data-prepper-plugins/opensearch/src/test/java/org/opensearch/dataprepper/plugins/sink/opensearch/index/DefaultIndexManagerTests.java

```diff
@@ -150,13 +150,17 @@ void getIndexAlias_IndexWithTimePattern(){
     }
 
     @Test
-    void getIndexAlias_IndexWithTimePattern_Exceptional_NotAsSuffix() {
-        when(indexConfiguration.getIndexAlias()).thenReturn(INDEX_ALIAS_WITH_TIME_PATTERN + "randomtext");
-        assertThrows(IllegalArgumentException.class,
-                () -> indexManagerFactory.getIndexManager(
-                        IndexType.CUSTOM, openSearchClient, restHighLevelClient, openSearchSinkConfiguration, templateStrategy));
-        verify(openSearchSinkConfiguration).getIndexConfiguration();
+    void getIndexAlias_IndexWithTimePattern_NotAsSuffix() {
+        when(indexConfiguration.getIndexAlias()).thenReturn(INDEX_ALIAS_WITH_TIME_PATTERN + "-randomtext");
+        defaultIndexManager = indexManagerFactory.getIndexManager(
+                IndexType.CUSTOM, openSearchClient, restHighLevelClient, openSearchSinkConfiguration, templateStrategy);
+        try {
+            final String indexAlias = defaultIndexManager.getIndexName(null);
+            assertThat(indexAlias, matchesPattern(Pattern.compile(INDEX_ALIAS + "-\\d{4}.\\d{2}.\\d{2}.\\d{2}-randomtext")));
+        } catch (IOException e){}
```

Comment:
Please do not add try-catch statements in unit tests. Just add ` throws IOException` to the method signature.

Reply:
Copied from another test. Fixed both.